framework/PostProcessors/CrossValidation.py

```diff
@@ -107,10 +108,18 @@ def _localReadMoreXML(self, xmlNode):
     #paramInput.parseNode(xmlNode)
 
     self.initializationOptionDict = {}
-
+    average = None
+    cvNode = xmlNode.find('SciKitLearn')
+    for child in cvNode:
+      if child.tag == 'average':
+        average = child
+        break
+    if average is not None:
+      self.averageScores = average.text.lower() in utils.stringsThatMeanTrue()
```

Comment:
Correct me if I'm wrong, but couldn't this be simplified to:
```python
if child.tag == 'average':
  self.averageScores = child.text.lower() in utils.stringsThatMeanTrue()
```
and eliminate the need for the `if average is not None` line?